code/modules/mob/living/simple_animal/abnormality/teth/spider_bud.dm

```diff
@@ -26,6 +26,10 @@
 	// If you do insight or have low prudence, fuck you and die for stepping on a spider
 	if(get_attribute_level(user, PRUDENCE_ATTRIBUTE) < 40 || work_type == ABNORMALITY_WORK_INSIGHT)
 		icon_state = "spider_open"
-		user.gib()
+		var/obj/structure/spider/cocoon/casing = new(user.loc)
```

Comment:
This will spawn cocoons on the same tile as the abnormality itself. Make it a step in random direction instead.